source/sandbox.rst

```diff
@@ -202,9 +202,15 @@ Read :ref:`postauth usage <postauth-usage>` for understanding how to fetch chall
       - Manual challenge with `transStatus` `Y` or `N`
       - `transStatus` `C` in `ARes` see `ARes outcome`_
 
-********
+*****
+Error
+*****
+
+If the last four digits do not match any of the given test cases above, an error will be given.
```

Comment:
I believe that this is only the case if a specific BIN range is used, right? As now only some ranges are ending towards our `standin` :shrug:

Reply:
All ranges are towards our standin.